webapp/src/ts/modules/contacts/contacts.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="filters">
+  <div class="inner">
+    <contacts-filters
+      (search)="search()"
+      (simIdentify)="simprintsIdentify()"
+      (sort)="sort($event)"
+      [simprintsEnabled]="simprintsEnabled"
+      [disabled]="false"
+      [sortDirection]="sortDirection"
+      [lastVisitedDateExtras]="lastVisitedDateExtras">
+    </contacts-filters>
+    <mm-navigation></mm-navigation>
+  </div>
+</div>
+
+<div class="inner">
+  <div class="inbox page">
+    <div id="contacts-list" class="col-sm-4 inbox-items left-pane">
+      <ul [hidden]="loading && !appending">
+        <mm-content-row
+          *ngFor="let contact of contactsList; trackBy: listTrackBy"
+          [id]="contact._id"
+          route="contacts"
+          [icon]="contact.icon"
+          [heading]="contact.name"
+          [valid]="contact.valid"
+          [summary]="contact.summary"
```

Comment:
You're missing a loooot of fields here. Please see https://github.com/medic/cht-core/blob/master/webapp/src/js/services/live-list.js#L88